### ISSUE TITLE
iOS 18 no option to close the mp4 video

### ISSUE THREAD

**Issue by kirito0206** [crazecoder/open_file#309]
After updating my device to iOS 18, an error appeared.
I am opening a video file. The video opens in full screen mode and start play. There are no video control buttons and no close button.
So it is not possible to go back to the apps.
Is there any suggestion what it could be?

Flutter 3.22.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 761747bfc5 (5 months ago) • 2024-06-05 22:15:13 +0200
Engine • revision edd8546116
Tools • Dart 3.4.3 • DevTools 2.34.3

Same issue as this address
https://github.com/crazecoder/open_file/issues/301

He removed the "audio" option in "UIBackgroundModes" to solve the problem, but I need this.

**Comment by kirito0206** [comment]
![image](https://github.com/user-attachments/assets/f5759651-34a6-4c91-80c6-4c9dd5a173f4)

**Comment by crazecoder** [comment]
I don't think this is a common issue caused by UIBackgroundModes on iOS 18. It may exist in a certain scenario. Unfortunately. This is the effect after I set UIBackgroundModes to audio.
![未命名](https://github.com/user-attachments/assets/a11666ab-0105-4450-96b4-3fd0adb8f433)

**Comment by kirito0206** [comment]
I have found the problem, it is caused by this animation.

On iOS 18, the navigation bar is not displayed due to the animation, and the playback page cannot be closed normally.
1. Add a delay solution
2. Turn off the animation (there is still no navigation bar on the first click)

https://github.com/javaherisaber/open_filex/pull/18
